models/collection.go

```diff
@@ -32,6 +32,10 @@ type Collection struct {
 	lazyLoad func(*Collection)
 }
 
+func (c *Collection) Key() string {
+	return c.key
```

Comment:
how about check whether key is nil or "" , if key is nil or "", we should  panic?

Reply:
this key is set when collection meta is list. will add check empty logic when use it's value